oak/server/enclave_server.h

```diff
@@ -168,11 +86,11 @@ class EnclaveServer final : public asylo::TrustedApplication {
   // The port on which the server is listening.
   int port_;
 
-  std::unique_ptr<::oak::OakNode> node_;
-  std::shared_ptr<::grpc::ServerCredentials> credentials_;
+  std::unique_ptr<OakNode> node_;
```

Comment:
@tiziano88 wanted to keep using ::oak:: for consistency, I think.  In general if we want to shorten these, I would prefer adding using directives.

Reply:
I don't particularly mind either way, but I would leave that change to a separate PR, perhaps at the same time as adding a "style guide" section to the README or CONTRIBUTING that explains what convention we are using going forward.

Reply:
I was trying to follow here totw/151 with qualifying namespaces. I'd suggest following established Google best practices unless there's a good reason to not to do so.

Reply:
Alright, totw/151 works for me.